dispatcher/DAQController.py

```diff
@@ -275,19 +279,29 @@ def ControlDetector(self, command, detector, force=False):
         if (dt > self.timeouts[command] and dt_last > self.time_between_commands) or force:
             run_mode = self.goal_state[detector]['mode']
             if command == 'arm':
+                if self.one_detector_arming:
+                    self.log.info('Another detector already arming, can\'t arm %s' % detector)
+                    # this leads to run number overlaps
+                    return
                 readers, cc = self.mongo.GetHostsForMode(run_mode)
                 hosts = (cc, readers)
                 delay = 0
+                self.one_detector_arming = True
             elif command == 'start':
                 readers, cc = self.mongo.GetHostsForMode(run_mode)
                 hosts = (readers, cc) # we want the cc to delay by 1s
-                delay = 1
```

Comment:
Was this supposed to be the delay of 1 sec between the START commands? Is this the setting to 0 instead of 1 sec delay which caused the problem during the weekend?